policies.tex

```diff
@@ -47,7 +47,7 @@ \subsection{}After electing all other candidates, Members-at-Large will be elect
 \subarticle{Appointments}
 
 \subsection{}The Executive Committee may appoint additional officers, including but not limited to those required by the constitution or standing policies.
-\subsection{}The term of an appointed officer shall not extend beyond the end of the term of the appointing EC.
+\subsection{}The term of an appointed officer shall not extend more than one month beyond the end of the term of the appointing EC, and shall lapse upon their replacement taking office.
```

Comment:
Another wording possibility, parallel to the "Officers shall take office on the earlier of one week after the spring class ends or June 1" in the constitution, is "The term of an appointed officer shall end on the earlier of when their replacement takes office or one month after the new EC takes office."